mods/ra/rules/player.yaml

```diff
@@ -18,6 +18,7 @@ Player:
 		LowPowerSlowdown: 3
 		QueuedAudio: Building
 		SpeedUp: True
+		BuildTimeSpeedReduction: 100, 75, 60, 50
```

Comment:
I can relate to what pchote is saying. For several months I was under the misunderstanding that production speed was a flat 7% per extra building up to 7 buildings. I didn’t realize that earlier buildings had more impact than late ones, which resulted in some poor theorycrafting towards my build order. Different rules could add some misunderstandings for players.